frontend/src/components/ThemedWindow.jsx

```diff
@@ -130,6 +147,7 @@ const ThemedWindow = ({ children, variant, slim, className: externClassName }) =
       [classes.gdq2]: search.has('gdq2'),
       [classes.gdq3]: search.has('gdq3'),
       [classes.avsr]: search.has('avsr'),
+      [classes.avsr]: search.has('noreset'),
```

Comment:
`classes.noreset` I think is what you want?

Reply:
fixed